src/main/java/com/google/neighborgood/helper/RetrieveUserInfo.java

```diff
@@ -17,24 +17,21 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Query.FilterOperator;
-import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
 import com.google.appengine.api.users.UserService;
 import java.util.ArrayList;
 import java.util.List;
 
 public final class RetrieveUserInfo {
```

Comment:
Can you add some unit tests for this class?

Reply:
Yep! Should I go ahead and add these into this PR or create a new one for it?

Reply:
Ahh I have already added a test case for this class in PR #79

Reply:
In PR #79 there is a java file called `HelperFunctionTest.java` that tested the `RetrieveUserInfo` class and `RewardingPoints` class

Reply:
Ah, got it!